src/ui/pages/search/SearchHeaderFilters.jsx

```diff
@@ -0,0 +1,55 @@
+import EAMCheckbox from "eam-components/dist/ui/components/inputs-ng/EAMCheckbox";
```

Comment:
FYI. Filters are not working right now, I'm going fix them shortly, it should be so hard 😄 But if You have time, check the other functionalities, thanks!